src/components/Attraction.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import mark from "../images/Location-mark.png"
```

Comment:
I can't run it locally, but your screenshot had the wrong path. This, however, is the **correct** path to the image and should work. 

If the image still won't display, please read [this](https://bobbyhadz.com/blog/react-import-image) article that explained clearly the different ways you can use images in your components.